## 09_fix_korean_fd_files.diff by <danai@debian.org>
##
## Fix texinput/KS/HLaTeX/*.fd files.
## The upstream package still uses the obsolete HLaTeX
## font packages, which is now superseded by ko.tex.
## The ko.tex fonts have not retained every font variant
## from HLaTeX, and some others were added.

Index: cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63bm.fd
===================================================================
--- cjk-4.8.3+git20090105.orig/texinput/KS/HLaTeX/c63bm.fd	2010-05-17 01:03:13.579377358 +0200
+++ cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63bm.fd	2010-05-17 01:03:16.000000000 +0200
@@ -26,12 +26,12 @@
 \DeclareFontShape{C63}{bm}{m}{sl}{<-> CJK * wbmo}{}
 \DeclareFontShape{C63}{bm}{m}{it}{<-> CJKssub * bm/m/sl}{}
 
-\DeclareFontShape{C63}{bm}{x}{n}{<-> CJK * wbmx}{}
-\DeclareFontShape{C63}{bm}{x}{sl}{<-> CJK * wbmxo}{}
+\DeclareFontShape{C63}{bm}{x}{n}{<-> CJKssub * bm/m/n}{}
+\DeclareFontShape{C63}{bm}{x}{sl}{<-> CJKssub * bm/m/sl}{}
 \DeclareFontShape{C63}{bm}{x}{it}{<-> CJKssub * bm/x/sl}{}
 
-\DeclareFontShape{C63}{bm}{ux}{n}{<-> CJK * wbmu}{}
-\DeclareFontShape{C63}{bm}{ux}{sl}{<-> CJK * wbmuo}{}
+\DeclareFontShape{C63}{bm}{ux}{n}{<-> CJKssub * bm/m/n}{}
+\DeclareFontShape{C63}{bm}{ux}{sl}{<-> CJKssub * bm/m/sl}{}
 \DeclareFontShape{C63}{bm}{ux}{it}{<-> CJKssub * bm/ux/sl}{}
 
 \DeclareFontShape{C63}{bm}{bc}{n}{<-> CJKb * wbmc}{\CJKbold}
@@ -42,12 +42,12 @@
 \DeclareFontShape{C63}{bm}{b}{sl}{<-> CJKb * wbmo}{\CJKbold}
 \DeclareFontShape{C63}{bm}{b}{it}{<-> CJKssub * bm/b/sl}{}
 
-\DeclareFontShape{C63}{bm}{bx}{n}{<-> CJKb * wbmx}{\CJKbold}
-\DeclareFontShape{C63}{bm}{bx}{sl}{<-> CJKb * wbmxo}{\CJKbold}
+\DeclareFontShape{C63}{bm}{bx}{n}{<-> CJKssub * bm/b/n}{}
+\DeclareFontShape{C63}{bm}{bx}{sl}{<-> CJKssub * bm/b/sl}{}
 \DeclareFontShape{C63}{bm}{bx}{it}{<-> CJKssub * bm/bx/sl}{}
 
-\DeclareFontShape{C63}{bm}{bux}{n}{<-> CJKb * wbmu}{\CJKbold}
-\DeclareFontShape{C63}{bm}{bux}{sl}{<-> CJKb * wbmuo}{\CJKbold}
+\DeclareFontShape{C63}{bm}{bux}{n}{<-> CJKssub * bm/b/n}{}
+\DeclareFontShape{C63}{bm}{bux}{sl}{<-> CJKssub * bm/b/sl}{}
 \DeclareFontShape{C63}{bm}{bux}{it}{<-> CJKssub * bm/bux/sl}{}
 
 \endinput
Index: cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63dn.fd
===================================================================
--- cjk-4.8.3+git20090105.orig/texinput/KS/HLaTeX/c63dn.fd	2010-05-17 01:03:13.587353412 +0200
+++ cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63dn.fd	2010-05-17 01:03:16.000000000 +0200
@@ -18,52 +18,52 @@
 
 \DeclareFontFamily{C63}{dn}{\hyphenchar \font\m@ne}
 
-\DeclareFontShape{C63}{dn}{c}{n}{<-> CJK * wdnc}{}
-\DeclareFontShape{C63}{dn}{c}{sl}{<-> CJK * wdnco}{}
+\DeclareFontShape{C63}{dn}{c}{n}{<-> CJKssub * dn/m/n}{}
+\DeclareFontShape{C63}{dn}{c}{sl}{<-> CJKssub * dn/b/n}{}
 \DeclareFontShape{C63}{dn}{c}{it}{<-> CJKssub * dn/c/sl}{}
 
 \DeclareFontShape{C63}{dn}{m}{n}{<-> CJK * wdn}{}
-\DeclareFontShape{C63}{dn}{m}{sl}{<-> CJK * wdno}{}
+\DeclareFontShape{C63}{dn}{m}{sl}{<-> CJKssub * dn/b/n}{}
 \DeclareFontShape{C63}{dn}{m}{it}{<-> CJKssub * dn/m/sl}{}
 
-\DeclareFontShape{C63}{dn}{x}{n}{<-> CJK * wdnx}{}
-\DeclareFontShape{C63}{dn}{x}{sl}{<-> CJK * wdnxo}{}
+\DeclareFontShape{C63}{dn}{x}{n}{<-> CJKssub * dn/m/n}{}
+\DeclareFontShape{C63}{dn}{x}{sl}{<-> CJKssub * dn/b/n}{}
 \DeclareFontShape{C63}{dn}{x}{it}{<-> CJKssub * dn/x/sl}{}
 
-\DeclareFontShape{C63}{dn}{ux}{n}{<-> CJK * wdnu}{}
-\DeclareFontShape{C63}{dn}{ux}{sl}{<-> CJK * wdnuo}{}
+\DeclareFontShape{C63}{dn}{ux}{n}{<-> CJKssub * dn/m/n}{}
+\DeclareFontShape{C63}{dn}{ux}{sl}{<-> CJKssub * dn/b/n}{}
 \DeclareFontShape{C63}{dn}{ux}{it}{<-> CJKssub * dn/ux/sl}{}
 
-\DeclareFontShape{C63}{dn}{bc}{n}{<-> CJK * wdnbc}{}
-\DeclareFontShape{C63}{dn}{bc}{sl}{<-> CJK * wdnbco}{}
+\DeclareFontShape{C63}{dn}{bc}{n}{<-> CJKssub * dn/b/n}{}
+\DeclareFontShape{C63}{dn}{bc}{sl}{<-> CJKssub * dn/b/n}{}
 \DeclareFontShape{C63}{dn}{bc}{it}{<-> CJKssub * dn/bc/sl}{}
 
 \DeclareFontShape{C63}{dn}{b}{n}{<-> CJK * wdnb}{}
-\DeclareFontShape{C63}{dn}{b}{sl}{<-> CJK * wdnbo}{}
+\DeclareFontShape{C63}{dn}{b}{sl}{<-> CJKssub * dn/b/n}{}
 \DeclareFontShape{C63}{dn}{b}{it}{<-> CJKssub * dn/b/sl}{}
 
-\DeclareFontShape{C63}{dn}{bx}{n}{<-> CJK * wdnbx}{}
-\DeclareFontShape{C63}{dn}{bx}{sl}{<-> CJK * wdnbxo}{}
+\DeclareFontShape{C63}{dn}{bx}{n}{<-> CJKssub * dn/b/n}{}
+\DeclareFontShape{C63}{dn}{bx}{sl}{<-> CJKssub * dn/b/n}{}
 \DeclareFontShape{C63}{dn}{bx}{it}{<-> CJKssub * dn/bx/sl}{}
 
-\DeclareFontShape{C63}{dn}{bux}{n}{<-> CJK * wdnbu}{}
-\DeclareFontShape{C63}{dn}{bux}{sl}{<-> CJK * wdnbuo}{}
+\DeclareFontShape{C63}{dn}{bux}{n}{<-> CJKssub * dn/b/n}{}
+\DeclareFontShape{C63}{dn}{bux}{sl}{<-> CJKssub * dn/b/n}{}
 \DeclareFontShape{C63}{dn}{bux}{it}{<-> CJKssub * dn/bux/sl}{}
 
-\DeclareFontShape{C63}{dn}{lc}{n}{<-> CJK * wdnlc}{}
-\DeclareFontShape{C63}{dn}{lc}{sl}{<-> CJK * wdnlco}{}
+\DeclareFontShape{C63}{dn}{lc}{n}{<-> CJKssub * dn/m/n}{}
+\DeclareFontShape{C63}{dn}{lc}{sl}{<-> CJKssub * dn/b/n}{}
 \DeclareFontShape{C63}{dn}{lc}{it}{<-> CJKssub * dn/lc/sl}{}
 
-\DeclareFontShape{C63}{dn}{l}{n}{<-> CJK * wdnl}{}
-\DeclareFontShape{C63}{dn}{l}{sl}{<-> CJK * wdnlo}{}
+\DeclareFontShape{C63}{dn}{l}{n}{<-> CJKssub * dn/m/n}{}
+\DeclareFontShape{C63}{dn}{l}{sl}{<-> CJKssub * dn/b/n}{}
 \DeclareFontShape{C63}{dn}{l}{it}{<-> CJKssub * dn/l/sl}{}
 
-\DeclareFontShape{C63}{dn}{lx}{n}{<-> CJK * wdnlx}{}
-\DeclareFontShape{C63}{dn}{lx}{sl}{<-> CJK * wdnlxo}{}
+\DeclareFontShape{C63}{dn}{lx}{n}{<-> CJKssub * dn/m/n}{}
+\DeclareFontShape{C63}{dn}{lx}{sl}{<-> CJKssub * dn/b/n}{}
 \DeclareFontShape{C63}{dn}{lx}{it}{<-> CJKssub * dn/lx/sl}{}
 
-\DeclareFontShape{C63}{dn}{lux}{n}{<-> CJK * wdnlu}{}
-\DeclareFontShape{C63}{dn}{lux}{sl}{<-> CJK * wdnluo}{}
+\DeclareFontShape{C63}{dn}{lux}{n}{<-> CJKssub * dn/m/n}{}
+\DeclareFontShape{C63}{dn}{lux}{sl}{<-> CJKssub * dn/b/n}{}
 \DeclareFontShape{C63}{dn}{lux}{it}{<-> CJKssub * dn/lux/sl}{}
 
 \endinput
Index: cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63gr.fd
===================================================================
--- cjk-4.8.3+git20090105.orig/texinput/KS/HLaTeX/c63gr.fd	2010-05-17 01:03:13.599352798 +0200
+++ cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63gr.fd	2010-05-17 01:03:16.000000000 +0200
@@ -26,12 +26,12 @@
 \DeclareFontShape{C63}{gr}{m}{sl}{<-> CJK * wgro}{}
 \DeclareFontShape{C63}{gr}{m}{it}{<-> CJKssub * gr/m/sl}{}
 
-\DeclareFontShape{C63}{gr}{x}{n}{<-> CJK * wgrx}{}
-\DeclareFontShape{C63}{gr}{x}{sl}{<-> CJK * wgrxo}{}
+\DeclareFontShape{C63}{gr}{x}{n}{<-> CJKssub * gr/m/n}{}
+\DeclareFontShape{C63}{gr}{x}{sl}{<-> CJKssub * gr/m/sl}{}
 \DeclareFontShape{C63}{gr}{x}{it}{<-> CJKssub * gr/x/sl}{}
 
-\DeclareFontShape{C63}{gr}{ux}{n}{<-> CJK * wgru}{}
-\DeclareFontShape{C63}{gr}{ux}{sl}{<-> CJK * wgruo}{}
+\DeclareFontShape{C63}{gr}{ux}{n}{<-> CJKssub * gr/m/n}{}
+\DeclareFontShape{C63}{gr}{ux}{sl}{<-> CJKssub * gr/m/sl}{}
 \DeclareFontShape{C63}{gr}{ux}{it}{<-> CJKssub * gr/ux/sl}{}
 
 \DeclareFontShape{C63}{gr}{bc}{n}{<-> CJK * wgrbc}{}
@@ -42,12 +42,12 @@
 \DeclareFontShape{C63}{gr}{b}{sl}{<-> CJK * wgrbo}{}
 \DeclareFontShape{C63}{gr}{b}{it}{<-> CJKssub * gr/b/sl}{}
 
-\DeclareFontShape{C63}{gr}{bx}{n}{<-> CJK * wgrbx}{}
-\DeclareFontShape{C63}{gr}{bx}{sl}{<-> CJK * wgrbxo}{}
+\DeclareFontShape{C63}{gr}{bx}{n}{<-> CJKssub * gr/b/n}{}
+\DeclareFontShape{C63}{gr}{bx}{sl}{<-> CJKssub * gr/b/sl}{}
 \DeclareFontShape{C63}{gr}{bx}{it}{<-> CJKssub * gr/bx/sl}{}
 
-\DeclareFontShape{C63}{gr}{bux}{n}{<-> CJK * wgrbu}{}
-\DeclareFontShape{C63}{gr}{bux}{sl}{<-> CJK * wgrbuo}{}
+\DeclareFontShape{C63}{gr}{bux}{n}{<-> CJKssub * gr/b/n}{}
+\DeclareFontShape{C63}{gr}{bux}{sl}{<-> CJKssub * gr/b/sl}{}
 \DeclareFontShape{C63}{gr}{bux}{it}{<-> CJKssub * gr/bux/sl}{}
 
 \endinput
Index: cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63gs.fd
===================================================================
--- cjk-4.8.3+git20090105.orig/texinput/KS/HLaTeX/c63gs.fd	2010-05-17 01:03:13.607352249 +0200
+++ cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63gs.fd	2010-05-17 01:03:16.000000000 +0200
@@ -18,36 +18,36 @@
 
 \DeclareFontFamily{C63}{gs}{\hyphenchar \font\m@ne}
 
-\DeclareFontShape{C63}{gs}{bc}{n}{<-> CJK * wgsc}{}
-\DeclareFontShape{C63}{gs}{bc}{sl}{<-> CJK * wgsco}{}
+\DeclareFontShape{C63}{gs}{bc}{n}{<-> CJKb * wgsc}{\CJKbold}
+\DeclareFontShape{C63}{gs}{bc}{sl}{<-> CJKb * wgsco}{\CJKbold}
 \DeclareFontShape{C63}{gs}{bc}{it}{<-> CJKssub * gs/bc/sl}{}
 
-\DeclareFontShape{C63}{gs}{b}{n}{<-> CJK * wgs}{}
-\DeclareFontShape{C63}{gs}{b}{sl}{<-> CJK * wgso}{}
+\DeclareFontShape{C63}{gs}{b}{n}{<-> CJKb * wgs}{\CJKbold}
+\DeclareFontShape{C63}{gs}{b}{sl}{<-> CJKb * wgso}{\CJKbold}
 \DeclareFontShape{C63}{gs}{b}{it}{<-> CJKssub * gs/b/sl}{}
 
-\DeclareFontShape{C63}{gs}{bx}{n}{<-> CJK * wgsx}{}
-\DeclareFontShape{C63}{gs}{bx}{sl}{<-> CJK * wgsxo}{}
+\DeclareFontShape{C63}{gs}{bx}{n}{<-> CJKssub * gs/b/n}{}
+\DeclareFontShape{C63}{gs}{bx}{sl}{<-> CJKssub * gs/b/sl}{}
 \DeclareFontShape{C63}{gs}{bx}{it}{<-> CJKssub * gs/bx/sl}{}
 
-\DeclareFontShape{C63}{gs}{bux}{n}{<-> CJK * wgsu}{}
-\DeclareFontShape{C63}{gs}{bux}{sl}{<-> CJK * wgsuo}{}
+\DeclareFontShape{C63}{gs}{bux}{n}{<-> CJKssub * gs/b/n}{}
+\DeclareFontShape{C63}{gs}{bux}{sl}{<-> CJKssub * gs/b/sl}{}
 \DeclareFontShape{C63}{gs}{bux}{it}{<-> CJKssub * gs/bux/sl}{}
 
-\DeclareFontShape{C63}{gs}{c}{n}{<-> CJKssub * gs/bc/n}{}
-\DeclareFontShape{C63}{gs}{c}{sl}{<-> CJKssub * gs/bc/sl}{}
+\DeclareFontShape{C63}{gs}{c}{n}{<-> CJK * wgsc}{}
+\DeclareFontShape{C63}{gs}{c}{sl}{<-> CJK * wgsco}{}
 \DeclareFontShape{C63}{gs}{c}{it}{<-> CJKssub * gs/bc/sl}{}
 
-\DeclareFontShape{C63}{gs}{m}{n}{<-> CJKssub * gs/b/n}{}
-\DeclareFontShape{C63}{gs}{m}{sl}{<-> CJKssub * gs/b/sl}{}
+\DeclareFontShape{C63}{gs}{m}{n}{<-> CJK * wgs}{}
+\DeclareFontShape{C63}{gs}{m}{sl}{<-> CJK * wgso}{}
 \DeclareFontShape{C63}{gs}{m}{it}{<-> CJKssub * gs/b/sl}{}
 
-\DeclareFontShape{C63}{gs}{x}{n}{<-> CJKssub * gs/bx/n}{}
-\DeclareFontShape{C63}{gs}{x}{sl}{<-> CJKssub * gs/bx/sl}{}
-\DeclareFontShape{C63}{gs}{x}{it}{<-> CJKssub * gs/bx/sl}{}
-
-\DeclareFontShape{C63}{gs}{ux}{n}{<-> CJKssub * gs/bux/n}{}
-\DeclareFontShape{C63}{gs}{ux}{sl}{<-> CJKssub * gs/bux/sl}{}
-\DeclareFontShape{C63}{gs}{ux}{it}{<-> CJKssub * gs/bux/sl}{}
+\DeclareFontShape{C63}{gs}{x}{n}{<-> CJKssub * gs/m/n}{}
+\DeclareFontShape{C63}{gs}{x}{sl}{<-> CJKssub * gs/m/sl}{}
+\DeclareFontShape{C63}{gs}{x}{it}{<-> CJKssub * gs/x/sl}{}
+
+\DeclareFontShape{C63}{gs}{ux}{n}{<-> CJKssub * gs/m/n}{}
+\DeclareFontShape{C63}{gs}{ux}{sl}{<-> CJKssub * gs/m/sl}{}
+\DeclareFontShape{C63}{gs}{ux}{it}{<-> CJKssub * gs/ux/sl}{}
 
 \endinput
Index: cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63gt.fd
===================================================================
--- cjk-4.8.3+git20090105.orig/texinput/KS/HLaTeX/c63gt.fd	2010-05-17 01:03:13.615353377 +0200
+++ cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63gt.fd	2010-05-17 01:03:16.000000000 +0200
@@ -26,12 +26,12 @@
 \DeclareFontShape{C63}{gt}{m}{sl}{<-> CJK * wgto}{}
 \DeclareFontShape{C63}{gt}{m}{it}{<-> CJKssub * gt/m/sl}{}
 
-\DeclareFontShape{C63}{gt}{x}{n}{<-> CJK * wgtx}{}
-\DeclareFontShape{C63}{gt}{x}{sl}{<-> CJK * wgtxo}{}
+\DeclareFontShape{C63}{gt}{x}{n}{<-> CJKssub * gt/m/n}{}
+\DeclareFontShape{C63}{gt}{x}{sl}{<-> CJKssub * gt/m/sl}{}
 \DeclareFontShape{C63}{gt}{x}{it}{<-> CJKssub * gt/x/sl}{}
 
-\DeclareFontShape{C63}{gt}{ux}{n}{<-> CJK * wgtu}{}
-\DeclareFontShape{C63}{gt}{ux}{sl}{<-> CJK * wgtuo}{}
+\DeclareFontShape{C63}{gt}{ux}{n}{<-> CJKssub * gt/m/n}{}
+\DeclareFontShape{C63}{gt}{ux}{sl}{<-> CJKssub * gt/m/sl}{}
 \DeclareFontShape{C63}{gt}{ux}{it}{<-> CJKssub * gt/ux/sl}{}
 
 \DeclareFontShape{C63}{gt}{bc}{n}{<-> CJK * wgtbc}{}
@@ -42,12 +42,12 @@
 \DeclareFontShape{C63}{gt}{b}{sl}{<-> CJK * wgtbo}{}
 \DeclareFontShape{C63}{gt}{b}{it}{<-> CJKssub * gt/b/sl}{}
 
-\DeclareFontShape{C63}{gt}{bx}{n}{<-> CJK * wgtbx}{}
-\DeclareFontShape{C63}{gt}{bx}{sl}{<-> CJK * wgtbxo}{}
+\DeclareFontShape{C63}{gt}{bx}{n}{<-> CJKssub * gt/b/n}{}
+\DeclareFontShape{C63}{gt}{bx}{sl}{<-> CJKssub * gt/b/sl}{}
 \DeclareFontShape{C63}{gt}{bx}{it}{<-> CJKssub * gt/bx/sl}{}
 
-\DeclareFontShape{C63}{gt}{bux}{n}{<-> CJK * wgtbu}{}
-\DeclareFontShape{C63}{gt}{bux}{sl}{<-> CJK * wgtbuo}{}
+\DeclareFontShape{C63}{gt}{bux}{n}{<-> CJKssub * gt/b/n}{}
+\DeclareFontShape{C63}{gt}{bux}{sl}{<-> CJKssub * gt/b/sl}{}
 \DeclareFontShape{C63}{gt}{bux}{it}{<-> CJKssub * gt/bux/sl}{}
 
 \endinput
Index: cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63jgt.fd
===================================================================
--- cjk-4.8.3+git20090105.orig/texinput/KS/HLaTeX/c63jgt.fd	2010-05-17 01:03:13.627353112 +0200
+++ cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63jgt.fd	2010-05-17 01:03:16.000000000 +0200
@@ -18,36 +18,36 @@
 
 \DeclareFontFamily{C63}{jgt}{\hyphenchar \font\m@ne}
 
-\DeclareFontShape{C63}{jgt}{c}{n}{<-> CJK * wjgtc}{}
-\DeclareFontShape{C63}{jgt}{c}{sl}{<-> CJK * wjgtco}{}
+\DeclareFontShape{C63}{jgt}{c}{n}{<-> CJKssub * jgt/m/n}{}
+\DeclareFontShape{C63}{jgt}{c}{sl}{<-> CJKssub * jgt/b/n}{}
 \DeclareFontShape{C63}{jgt}{c}{it}{<-> CJKssub * jgt/c/sl}{}
 
 \DeclareFontShape{C63}{jgt}{m}{n}{<-> CJK * wjgt}{}
-\DeclareFontShape{C63}{jgt}{m}{sl}{<-> CJK * wjgto}{}
+\DeclareFontShape{C63}{jgt}{m}{sl}{<-> CJKssub * jgt/b/n}{}
 \DeclareFontShape{C63}{jgt}{m}{it}{<-> CJKssub * jgt/m/sl}{}
 
-\DeclareFontShape{C63}{jgt}{x}{n}{<-> CJK * wjgtx}{}
-\DeclareFontShape{C63}{jgt}{x}{sl}{<-> CJK * wjgtxo}{}
+\DeclareFontShape{C63}{jgt}{x}{n}{<-> CJKssub * jgt/m/n}{}
+\DeclareFontShape{C63}{jgt}{x}{sl}{<-> CJKssub * jgt/b/n}{}
 \DeclareFontShape{C63}{jgt}{x}{it}{<-> CJKssub * jgt/x/sl}{}
 
-\DeclareFontShape{C63}{jgt}{ux}{n}{<-> CJK * wjgtu}{}
-\DeclareFontShape{C63}{jgt}{ux}{sl}{<-> CJK * wjgtuo}{}
+\DeclareFontShape{C63}{jgt}{ux}{n}{<-> CJKssub * jgt/m/n}{}
+\DeclareFontShape{C63}{jgt}{ux}{sl}{<-> CJKssub * jgt/b/n}{}
 \DeclareFontShape{C63}{jgt}{ux}{it}{<-> CJKssub * jgt/ux/sl}{}
 
-\DeclareFontShape{C63}{jgt}{bc}{n}{<-> CJKb * wjgtc}{\CJKbold}
-\DeclareFontShape{C63}{jgt}{bc}{sl}{<-> CJKb * wjgtco}{\CJKbold}
+\DeclareFontShape{C63}{jgt}{bc}{n}{<-> CJKssub * jgt/b/n}{}
+\DeclareFontShape{C63}{jgt}{bc}{sl}{<-> CJKssub * jgt/b/n}{}
 \DeclareFontShape{C63}{jgt}{bc}{it}{<-> CJKssub * jgt/bc/sl}{}
 
 \DeclareFontShape{C63}{jgt}{b}{n}{<-> CJKb * wjgt}{\CJKbold}
-\DeclareFontShape{C63}{jgt}{b}{sl}{<-> CJKb * wjgto}{\CJKbold}
+\DeclareFontShape{C63}{jgt}{b}{sl}{<-> CJKssub * jgt/b/n}{}
 \DeclareFontShape{C63}{jgt}{b}{it}{<-> CJKssub * jgt/b/sl}{}
 
-\DeclareFontShape{C63}{jgt}{bx}{n}{<-> CJKb * wjgtx}{\CJKbold}
-\DeclareFontShape{C63}{jgt}{bx}{sl}{<-> CJKb * wjgtxo}{\CJKbold}
+\DeclareFontShape{C63}{jgt}{bx}{n}{<-> CJKssub * jgt/b/n}{}
+\DeclareFontShape{C63}{jgt}{bx}{sl}{<-> CJKssub * jgt/b/n}{}
 \DeclareFontShape{C63}{jgt}{bx}{it}{<-> CJKssub * jgt/bx/sl}{}
 
-\DeclareFontShape{C63}{jgt}{bux}{n}{<-> CJKb * wjgtu}{\CJKbold}
-\DeclareFontShape{C63}{jgt}{bux}{sl}{<-> CJKb * wjgtuo}{\CJKbold}
+\DeclareFontShape{C63}{jgt}{bux}{n}{<-> CJKssub * jgt/b/n}{}
+\DeclareFontShape{C63}{jgt}{bux}{sl}{<-> CJKssub * jgt/b/n}{}
 \DeclareFontShape{C63}{jgt}{bux}{it}{<-> CJKssub * jgt/bux/sl}{}
 
 \endinput
Index: cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63jmj.fd
===================================================================
--- cjk-4.8.3+git20090105.orig/texinput/KS/HLaTeX/c63jmj.fd	2010-05-17 01:03:13.639352847 +0200
+++ cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63jmj.fd	2010-05-17 01:03:16.000000000 +0200
@@ -18,36 +18,36 @@
 
 \DeclareFontFamily{C63}{jmj}{\hyphenchar \font\m@ne}
 
-\DeclareFontShape{C63}{jmj}{c}{n}{<-> CJK * wjmjc}{}
-\DeclareFontShape{C63}{jmj}{c}{sl}{<-> CJK * wjmjco}{}
+\DeclareFontShape{C63}{jmj}{c}{n}{<-> CJKssub * jmj/m/n}{}
+\DeclareFontShape{C63}{jmj}{c}{sl}{<-> CJKssub * jmj/m/n}{}
 \DeclareFontShape{C63}{jmj}{c}{it}{<-> CJKssub * jmj/c/sl}{}
 
 \DeclareFontShape{C63}{jmj}{m}{n}{<-> CJK * wjmj}{}
-\DeclareFontShape{C63}{jmj}{m}{sl}{<-> CJK * wjmjo}{}
+\DeclareFontShape{C63}{jmj}{m}{sl}{<-> CJKssub * jmj/m/n}{}
 \DeclareFontShape{C63}{jmj}{m}{it}{<-> CJKssub * jmj/m/sl}{}
 
-\DeclareFontShape{C63}{jmj}{x}{n}{<-> CJK * wjmjx}{}
-\DeclareFontShape{C63}{jmj}{x}{sl}{<-> CJK * wjmjxo}{}
+\DeclareFontShape{C63}{jmj}{x}{n}{<-> CJKssub * jmj/m/n}{}
+\DeclareFontShape{C63}{jmj}{x}{sl}{<-> CJKssub * jmj/m/n}{}
 \DeclareFontShape{C63}{jmj}{x}{it}{<-> CJKssub * jmj/x/sl}{}
 
-\DeclareFontShape{C63}{jmj}{ux}{n}{<-> CJK * wjmju}{}
-\DeclareFontShape{C63}{jmj}{ux}{sl}{<-> CJK * wjmjuo}{}
+\DeclareFontShape{C63}{jmj}{ux}{n}{<-> CJKssub * jmj/m/n}{}
+\DeclareFontShape{C63}{jmj}{ux}{sl}{<-> CJKssub * jmj/m/n}{}
 \DeclareFontShape{C63}{jmj}{ux}{it}{<-> CJKssub * jmj/ux/sl}{}
 
-\DeclareFontShape{C63}{jmj}{bc}{n}{<-> CJKb * wjmjc}{\CJKbold}
-\DeclareFontShape{C63}{jmj}{bc}{sl}{<-> CJKb * wjmjco}{\CJKbold}
+\DeclareFontShape{C63}{jmj}{bc}{n}{<-> CJKssub * jmj/b/n}{}
+\DeclareFontShape{C63}{jmj}{bc}{sl}{<-> CJKssub * jmj/b/n}{}
 \DeclareFontShape{C63}{jmj}{bc}{it}{<-> CJKssub * jmj/bc/sl}{}
 
 \DeclareFontShape{C63}{jmj}{b}{n}{<-> CJKb * wjmj}{\CJKbold}
-\DeclareFontShape{C63}{jmj}{b}{sl}{<-> CJKb * wjmjo}{\CJKbold}
+\DeclareFontShape{C63}{jmj}{b}{sl}{<-> CJKssub * jmj/b/n}{}
 \DeclareFontShape{C63}{jmj}{b}{it}{<-> CJKssub * jmj/b/sl}{}
 
-\DeclareFontShape{C63}{jmj}{bx}{n}{<-> CJKb * wjmjx}{\CJKbold}
-\DeclareFontShape{C63}{jmj}{bx}{sl}{<-> CJKb * wjmjxo}{\CJKbold}
+\DeclareFontShape{C63}{jmj}{bx}{n}{<-> CJKssub * jmj/b/n}{}
+\DeclareFontShape{C63}{jmj}{bx}{sl}{<-> CJKssub * jmj/b/n}{}
 \DeclareFontShape{C63}{jmj}{bx}{it}{<-> CJKssub * jmj/bx/sl}{}
 
-\DeclareFontShape{C63}{jmj}{bux}{n}{<-> CJKb * wjmju}{\CJKbold}
-\DeclareFontShape{C63}{jmj}{bux}{sl}{<-> CJKb * wjmjuo}{\CJKbold}
+\DeclareFontShape{C63}{jmj}{bux}{n}{<-> CJKssub * jmj/b/n}{}
+\DeclareFontShape{C63}{jmj}{bux}{sl}{<-> CJKssub * jmj/b/n}{}
 \DeclareFontShape{C63}{jmj}{bux}{it}{<-> CJKssub * jmj/bux/sl}{}
 
 \endinput
Index: cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63jnv.fd
===================================================================
--- cjk-4.8.3+git20090105.orig/texinput/KS/HLaTeX/c63jnv.fd	2010-05-17 01:03:13.647356279 +0200
+++ cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63jnv.fd	2010-05-17 01:03:16.000000000 +0200
@@ -18,36 +18,36 @@
 
 \DeclareFontFamily{C63}{jnv}{\hyphenchar \font\m@ne}
 
-\DeclareFontShape{C63}{jnv}{c}{n}{<-> CJK * wjnvc}{}
-\DeclareFontShape{C63}{jnv}{c}{sl}{<-> CJK * wjnvco}{}
+\DeclareFontShape{C63}{jnv}{c}{n}{<-> CJKssub * jnv/m/n}{}
+\DeclareFontShape{C63}{jnv}{c}{sl}{<-> CJKssub * jnv/b/n}{}
 \DeclareFontShape{C63}{jnv}{c}{it}{<-> CJKssub * jnv/c/sl}{}
 
 \DeclareFontShape{C63}{jnv}{m}{n}{<-> CJK * wjnv}{}
-\DeclareFontShape{C63}{jnv}{m}{sl}{<-> CJK * wjnvo}{}
+\DeclareFontShape{C63}{jnv}{m}{sl}{<-> CJKssub * jnv/b/n}{}
 \DeclareFontShape{C63}{jnv}{m}{it}{<-> CJKssub * jnv/m/sl}{}
 
-\DeclareFontShape{C63}{jnv}{x}{n}{<-> CJK * wjnvx}{}
-\DeclareFontShape{C63}{jnv}{x}{sl}{<-> CJK * wjnvxo}{}
+\DeclareFontShape{C63}{jnv}{x}{n}{<-> CJKssub * jnv/m/n}{}
+\DeclareFontShape{C63}{jnv}{x}{sl}{<-> CJKssub * jnv/b/n}{}
 \DeclareFontShape{C63}{jnv}{x}{it}{<-> CJKssub * jnv/x/sl}{}
 
-\DeclareFontShape{C63}{jnv}{ux}{n}{<-> CJK * wjnvu}{}
-\DeclareFontShape{C63}{jnv}{ux}{sl}{<-> CJK * wjnvuo}{}
+\DeclareFontShape{C63}{jnv}{ux}{n}{<-> CJKssub * jnv/m/n}{}
+\DeclareFontShape{C63}{jnv}{ux}{sl}{<-> CJKssub * jnv/b/n}{}
 \DeclareFontShape{C63}{jnv}{ux}{it}{<-> CJKssub * jnv/ux/sl}{}
 
-\DeclareFontShape{C63}{jnv}{bc}{n}{<-> CJKb * wjnvc}{\CJKbold}
-\DeclareFontShape{C63}{jnv}{bc}{sl}{<-> CJKb * wjnvco}{\CJKbold}
+\DeclareFontShape{C63}{jnv}{bc}{n}{<-> CJKssub * jnv/b/n}{}
+\DeclareFontShape{C63}{jnv}{bc}{sl}{<-> CJKssub * jnv/b/n}{}
 \DeclareFontShape{C63}{jnv}{bc}{it}{<-> CJKssub * jnv/bc/sl}{}
 
 \DeclareFontShape{C63}{jnv}{b}{n}{<-> CJKb * wjnv}{\CJKbold}
-\DeclareFontShape{C63}{jnv}{b}{sl}{<-> CJKb * wjnvo}{\CJKbold}
+\DeclareFontShape{C63}{jnv}{b}{sl}{<-> CJKssub * jnv/b/n}{}
 \DeclareFontShape{C63}{jnv}{b}{it}{<-> CJKssub * jnv/b/sl}{}
 
-\DeclareFontShape{C63}{jnv}{bx}{n}{<-> CJKb * wjnvx}{\CJKbold}
-\DeclareFontShape{C63}{jnv}{bx}{sl}{<-> CJKb * wjnvxo}{\CJKbold}
+\DeclareFontShape{C63}{jnv}{bx}{n}{<-> CJKssub * jnv/b/n}{}
+\DeclareFontShape{C63}{jnv}{bx}{sl}{<-> CJKssub * jnv/b/n}{}
 \DeclareFontShape{C63}{jnv}{bx}{it}{<-> CJKssub * jnv/bx/sl}{}
 
-\DeclareFontShape{C63}{jnv}{bux}{n}{<-> CJKb * wjnvu}{\CJKbold}
-\DeclareFontShape{C63}{jnv}{bux}{sl}{<-> CJKb * wjnvuo}{\CJKbold}
+\DeclareFontShape{C63}{jnv}{bux}{n}{<-> CJKssub * jnv/b/n}{}
+\DeclareFontShape{C63}{jnv}{bux}{sl}{<-> CJKssub * jnv/b/n}{}
 \DeclareFontShape{C63}{jnv}{bux}{it}{<-> CJKssub * jnv/bux/sl}{}
 
 \endinput
Index: cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63jsr.fd
===================================================================
--- cjk-4.8.3+git20090105.orig/texinput/KS/HLaTeX/c63jsr.fd	2010-05-17 01:03:13.655352866 +0200
+++ cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63jsr.fd	2010-05-17 01:03:16.000000000 +0200
@@ -18,36 +18,36 @@
 
 \DeclareFontFamily{C63}{jsr}{\hyphenchar \font\m@ne}
 
-\DeclareFontShape{C63}{jsr}{c}{n}{<-> CJK * wjsrc}{}
-\DeclareFontShape{C63}{jsr}{c}{sl}{<-> CJK * wjsrco}{}
+\DeclareFontShape{C63}{jsr}{c}{n}{<-> CJKssub * jsr/m/n}{}
+\DeclareFontShape{C63}{jsr}{c}{sl}{<-> CJKssub * jsr/b/n}{}
 \DeclareFontShape{C63}{jsr}{c}{it}{<-> CJKssub * jsr/c/sl}{}
 
 \DeclareFontShape{C63}{jsr}{m}{n}{<-> CJK * wjsr}{}
-\DeclareFontShape{C63}{jsr}{m}{sl}{<-> CJK * wjsro}{}
+\DeclareFontShape{C63}{jsr}{m}{sl}{<-> CJKssub * jsr/b/n}{}
 \DeclareFontShape{C63}{jsr}{m}{it}{<-> CJKssub * jsr/m/sl}{}
 
-\DeclareFontShape{C63}{jsr}{x}{n}{<-> CJK * wjsrx}{}
-\DeclareFontShape{C63}{jsr}{x}{sl}{<-> CJK * wjsrxo}{}
+\DeclareFontShape{C63}{jsr}{x}{n}{<-> CJKssub * jsr/m/n}{}
+\DeclareFontShape{C63}{jsr}{x}{sl}{<-> CJKssub * jsr/b/n}{}
 \DeclareFontShape{C63}{jsr}{x}{it}{<-> CJKssub * jsr/x/sl}{}
 
-\DeclareFontShape{C63}{jsr}{ux}{n}{<-> CJK * wjsru}{}
-\DeclareFontShape{C63}{jsr}{ux}{sl}{<-> CJK * wjsruo}{}
+\DeclareFontShape{C63}{jsr}{ux}{n}{<-> CJKssub * jsr/m/n}{}
+\DeclareFontShape{C63}{jsr}{ux}{sl}{<-> CJKssub * jsr/b/n}{}
 \DeclareFontShape{C63}{jsr}{ux}{it}{<-> CJKssub * jsr/ux/sl}{}
 
-\DeclareFontShape{C63}{jsr}{bc}{n}{<-> CJKb * wjsrc}{\CJKbold}
-\DeclareFontShape{C63}{jsr}{bc}{sl}{<-> CJKb * wjsrco}{\CJKbold}
+\DeclareFontShape{C63}{jsr}{bc}{n}{<-> CJKssub * jsr/b/n}{}
+\DeclareFontShape{C63}{jsr}{bc}{sl}{<-> CJKssub * jsr/b/n}{}
 \DeclareFontShape{C63}{jsr}{bc}{it}{<-> CJKssub * jsr/bc/sl}{}
 
 \DeclareFontShape{C63}{jsr}{b}{n}{<-> CJKb * wjsr}{\CJKbold}
-\DeclareFontShape{C63}{jsr}{b}{sl}{<-> CJKb * wjsro}{\CJKbold}
+\DeclareFontShape{C63}{jsr}{b}{sl}{<-> CJKssub * jsr/b/n}{}
 \DeclareFontShape{C63}{jsr}{b}{it}{<-> CJKssub * jsr/b/sl}{}
 
-\DeclareFontShape{C63}{jsr}{bx}{n}{<-> CJKb * wjsrx}{\CJKbold}
-\DeclareFontShape{C63}{jsr}{bx}{sl}{<-> CJKb * wjsrxo}{\CJKbold}
+\DeclareFontShape{C63}{jsr}{bx}{n}{<-> CJKssub * jsr/b/n}{}
+\DeclareFontShape{C63}{jsr}{bx}{sl}{<-> CJKssub * jsr/b/n}{}
 \DeclareFontShape{C63}{jsr}{bx}{it}{<-> CJKssub * jsr/bx/sl}{}
 
-\DeclareFontShape{C63}{jsr}{bux}{n}{<-> CJKb * wjsru}{\CJKbold}
-\DeclareFontShape{C63}{jsr}{bux}{sl}{<-> CJKb * wjsruo}{\CJKbold}
+\DeclareFontShape{C63}{jsr}{bux}{n}{<-> CJKssub * jsr/b/n}{}
+\DeclareFontShape{C63}{jsr}{bux}{sl}{<-> CJKssub * jsr/b/n}{}
 \DeclareFontShape{C63}{jsr}{bux}{it}{<-> CJKssub * jsr/bux/sl}{}
 
 \endinput
Index: cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63mj.fd
===================================================================
--- cjk-4.8.3+git20090105.orig/texinput/KS/HLaTeX/c63mj.fd	2010-05-17 01:03:13.667353019 +0200
+++ cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63mj.fd	2010-05-17 01:03:16.000000000 +0200
@@ -26,12 +26,12 @@
 \DeclareFontShape{C63}{mj}{m}{sl}{<-> CJK * wmjo}{}
 \DeclareFontShape{C63}{mj}{m}{it}{<-> CJKssub * mj/m/sl}{}
 
-\DeclareFontShape{C63}{mj}{x}{n}{<-> CJK * wmjx}{}
-\DeclareFontShape{C63}{mj}{x}{sl}{<-> CJK * wmjxo}{}
+\DeclareFontShape{C63}{mj}{x}{n}{<-> CJKssub * mj/m/n}{}
+\DeclareFontShape{C63}{mj}{x}{sl}{<-> CJKssub * mj/m/sl}{}
 \DeclareFontShape{C63}{mj}{x}{it}{<-> CJKssub * mj/x/sl}{}
 
-\DeclareFontShape{C63}{mj}{ux}{n}{<-> CJK * wmju}{}
-\DeclareFontShape{C63}{mj}{ux}{sl}{<-> CJK * wmjuo}{}
+\DeclareFontShape{C63}{mj}{ux}{n}{<-> CJKssub * mj/m/n}{}
+\DeclareFontShape{C63}{mj}{ux}{sl}{<-> CJKssub * mj/m/sl}{}
 \DeclareFontShape{C63}{mj}{ux}{it}{<-> CJKssub * mj/ux/sl}{}
 
 \DeclareFontShape{C63}{mj}{bc}{n}{<-> CJK * wmjbc}{}
@@ -42,12 +42,12 @@
 \DeclareFontShape{C63}{mj}{b}{sl}{<-> CJK * wmjbo}{}
 \DeclareFontShape{C63}{mj}{b}{it}{<-> CJKssub * mj/b/sl}{}
 
-\DeclareFontShape{C63}{mj}{bx}{n}{<-> CJK * wmjbx}{}
-\DeclareFontShape{C63}{mj}{bx}{sl}{<-> CJK * wmjbxo}{}
+\DeclareFontShape{C63}{mj}{bx}{n}{<-> CJKssub * mj/b/n}{}
+\DeclareFontShape{C63}{mj}{bx}{sl}{<-> CJKssub * mj/b/sl}{}
 \DeclareFontShape{C63}{mj}{bx}{it}{<-> CJKssub * mj/bx/sl}{}
 
-\DeclareFontShape{C63}{mj}{bux}{n}{<-> CJK * wmjbu}{}
-\DeclareFontShape{C63}{mj}{bux}{sl}{<-> CJK * wmjbuo}{}
+\DeclareFontShape{C63}{mj}{bux}{n}{<-> CJKssub * mj/b/n}{}
+\DeclareFontShape{C63}{mj}{bux}{sl}{<-> CJKssub * mj/b/sl}{}
 \DeclareFontShape{C63}{mj}{bux}{it}{<-> CJKssub * mj/bux/sl}{}
 
 \endinput
Index: cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63pga.fd
===================================================================
--- cjk-4.8.3+git20090105.orig/texinput/KS/HLaTeX/c63pga.fd	2010-05-17 01:03:13.675353518 +0200
+++ cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63pga.fd	2010-05-17 01:03:16.000000000 +0200
@@ -18,36 +18,36 @@
 
 \DeclareFontFamily{C63}{pga}{\hyphenchar \font\m@ne}
 
-\DeclareFontShape{C63}{pga}{c}{n}{<-> CJK * wpgac}{}
-\DeclareFontShape{C63}{pga}{c}{sl}{<-> CJK * wpgaco}{}
+\DeclareFontShape{C63}{pga}{c}{n}{<-> CJKssub * pga/m/n}{}
+\DeclareFontShape{C63}{pga}{c}{sl}{<-> CJKssub * pga/b/n}{}
 \DeclareFontShape{C63}{pga}{c}{it}{<-> CJKssub * pga/c/sl}{}
 
 \DeclareFontShape{C63}{pga}{m}{n}{<-> CJK * wpga}{}
-\DeclareFontShape{C63}{pga}{m}{sl}{<-> CJK * wpgao}{}
+\DeclareFontShape{C63}{pga}{m}{sl}{<-> CJKssub * pga/b/n}{}
 \DeclareFontShape{C63}{pga}{m}{it}{<-> CJKssub * pga/m/sl}{}
 
-\DeclareFontShape{C63}{pga}{x}{n}{<-> CJK * wpgax}{}
-\DeclareFontShape{C63}{pga}{x}{sl}{<-> CJK * wpgaxo}{}
+\DeclareFontShape{C63}{pga}{x}{n}{<-> CJKssub * pga/m/n}{}
+\DeclareFontShape{C63}{pga}{x}{sl}{<-> CJKssub * pga/b/n}{}
 \DeclareFontShape{C63}{pga}{x}{it}{<-> CJKssub * pga/x/sl}{}
 
-\DeclareFontShape{C63}{pga}{ux}{n}{<-> CJK * wpgau}{}
-\DeclareFontShape{C63}{pga}{ux}{sl}{<-> CJK * wpgauo}{}
+\DeclareFontShape{C63}{pga}{ux}{n}{<-> CJKssub * pga/m/n}{}
+\DeclareFontShape{C63}{pga}{ux}{sl}{<-> CJKssub * pga/b/n}{}
 \DeclareFontShape{C63}{pga}{ux}{it}{<-> CJKssub * pga/ux/sl}{}
 
-\DeclareFontShape{C63}{pga}{bc}{n}{<-> CJKb * wpgac}{\CJKbold}
-\DeclareFontShape{C63}{pga}{bc}{sl}{<-> CJKb * wpgaco}{\CJKbold}
+\DeclareFontShape{C63}{pga}{bc}{n}{<-> CJKssub * pga/b/n}{}
+\DeclareFontShape{C63}{pga}{bc}{sl}{<-> CJKssub * pga/b/n}{}
 \DeclareFontShape{C63}{pga}{bc}{it}{<-> CJKssub * pga/bc/sl}{}
 
 \DeclareFontShape{C63}{pga}{b}{n}{<-> CJKb * wpga}{\CJKbold}
-\DeclareFontShape{C63}{pga}{b}{sl}{<-> CJKb * wpgao}{\CJKbold}
+\DeclareFontShape{C63}{pga}{b}{sl}{<-> CJKssub * pga/b/n}{}
 \DeclareFontShape{C63}{pga}{b}{it}{<-> CJKssub * pga/b/sl}{}
 
-\DeclareFontShape{C63}{pga}{bx}{n}{<-> CJKb * wpgax}{\CJKbold}
-\DeclareFontShape{C63}{pga}{bx}{sl}{<-> CJKb * wpgaxo}{\CJKbold}
+\DeclareFontShape{C63}{pga}{bx}{n}{<-> CJKssub * pga/b/n}{}
+\DeclareFontShape{C63}{pga}{bx}{sl}{<-> CJKssub * pga/b/n}{}
 \DeclareFontShape{C63}{pga}{bx}{it}{<-> CJKssub * pga/bx/sl}{}
 
-\DeclareFontShape{C63}{pga}{bux}{n}{<-> CJKb * wpgau}{\CJKbold}
-\DeclareFontShape{C63}{pga}{bux}{sl}{<-> CJKb * wpgauo}{\CJKbold}
+\DeclareFontShape{C63}{pga}{bux}{n}{<-> CJKssub * pga/b/n}{}
+\DeclareFontShape{C63}{pga}{bux}{sl}{<-> CJKssub * pga/b/n}{}
 \DeclareFontShape{C63}{pga}{bux}{it}{<-> CJKssub * pga/bux/sl}{}
 
 \endinput
Index: cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63pg.fd
===================================================================
--- cjk-4.8.3+git20090105.orig/texinput/KS/HLaTeX/c63pg.fd	2010-05-17 01:03:13.683353248 +0200
+++ cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63pg.fd	2010-05-17 01:03:16.000000000 +0200
@@ -20,34 +20,34 @@
 
 \DeclareFontShape{C63}{pg}{c}{n}{<-> CJK * wpgc}{}
 \DeclareFontShape{C63}{pg}{c}{sl}{<-> CJK * wpgco}{}
-\DeclareFontShape{C63}{pg}{c}{it}{<-> CJKssub * wwpg/c/sl}{}
+\DeclareFontShape{C63}{pg}{c}{it}{<-> CJKssub * pg/c/sl}{}
 
 \DeclareFontShape{C63}{pg}{m}{n}{<-> CJK * wpg}{}
 \DeclareFontShape{C63}{pg}{m}{sl}{<-> CJK * wpgo}{}
-\DeclareFontShape{C63}{pg}{m}{it}{<-> CJKssub * wwpg/m/sl}{}
+\DeclareFontShape{C63}{pg}{m}{it}{<-> CJKssub * pg/m/sl}{}
 
-\DeclareFontShape{C63}{pg}{x}{n}{<-> CJK * wpgx}{}
-\DeclareFontShape{C63}{pg}{x}{sl}{<-> CJK * wpgxo}{}
-\DeclareFontShape{C63}{pg}{x}{it}{<-> CJKssub * wwpg/x/sl}{}
-
-\DeclareFontShape{C63}{pg}{ux}{n}{<-> CJK * wpgu}{}
-\DeclareFontShape{C63}{pg}{ux}{sl}{<-> CJK * wpguo}{}
-\DeclareFontShape{C63}{pg}{ux}{it}{<-> CJKssub * wwpg/ux/sl}{}
+\DeclareFontShape{C63}{pg}{x}{n}{<-> CJKssub * pg/m/n}{}
+\DeclareFontShape{C63}{pg}{x}{sl}{<-> CJKssub * pg/m/sl}{}
+\DeclareFontShape{C63}{pg}{x}{it}{<-> CJKssub * pg/x/sl}{}
+
+\DeclareFontShape{C63}{pg}{ux}{n}{<-> CJKssub * pg/m/n}{}
+\DeclareFontShape{C63}{pg}{ux}{sl}{<-> CJKssub * pg/m/sl}{}
+\DeclareFontShape{C63}{pg}{ux}{it}{<-> CJKssub * pg/ux/sl}{}
 
 \DeclareFontShape{C63}{pg}{bc}{n}{<-> CJK * wpgbc}{}
 \DeclareFontShape{C63}{pg}{bc}{sl}{<-> CJK * wpgbco}{}
-\DeclareFontShape{C63}{pg}{bc}{it}{<-> CJKssub * wwpg/bc/sl}{}
+\DeclareFontShape{C63}{pg}{bc}{it}{<-> CJKssub * pg/bc/sl}{}
 
 \DeclareFontShape{C63}{pg}{b}{n}{<-> CJK * wpgb}{}
 \DeclareFontShape{C63}{pg}{b}{sl}{<-> CJK * wpgbo}{}
-\DeclareFontShape{C63}{pg}{b}{it}{<-> CJKssub * wwpg/b/sl}{}
+\DeclareFontShape{C63}{pg}{b}{it}{<-> CJKssub * pg/b/sl}{}
 
-\DeclareFontShape{C63}{pg}{bx}{n}{<-> CJK * wpgbx}{}
-\DeclareFontShape{C63}{pg}{bx}{sl}{<-> CJK * wpgbxo}{}
-\DeclareFontShape{C63}{pg}{bx}{it}{<-> CJKssub * wwpg/bx/sl}{}
-
-\DeclareFontShape{C63}{pg}{bux}{n}{<-> CJK * wpgbu}{}
-\DeclareFontShape{C63}{pg}{bux}{sl}{<-> CJK * wpgbuo}{}
-\DeclareFontShape{C63}{pg}{bux}{it}{<-> CJKssub * wwpg/bux/sl}{}
+\DeclareFontShape{C63}{pg}{bx}{n}{<-> CJKssub * pg/b/n}{}
+\DeclareFontShape{C63}{pg}{bx}{sl}{<-> CJKssub * pg/b/sl}{}
+\DeclareFontShape{C63}{pg}{bx}{it}{<-> CJKssub * pg/bx/sl}{}
+
+\DeclareFontShape{C63}{pg}{bux}{n}{<-> CJKssub * pg/b/n}{}
+\DeclareFontShape{C63}{pg}{bux}{sl}{<-> CJKssub * pg/b/sl}{}
+\DeclareFontShape{C63}{pg}{bux}{it}{<-> CJKssub * pg/bux/sl}{}
 
 \endinput
Index: cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63ph.fd
===================================================================
--- cjk-4.8.3+git20090105.orig/texinput/KS/HLaTeX/c63ph.fd	2010-05-17 01:03:13.691367504 +0200
+++ cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63ph.fd	2010-05-17 01:03:16.000000000 +0200
@@ -18,36 +18,36 @@
 
 \DeclareFontFamily{C63}{ph}{\hyphenchar \font\m@ne}
 
-\DeclareFontShape{C63}{ph}{c}{n}{<-> CJK * wphc}{}
-\DeclareFontShape{C63}{ph}{c}{sl}{<-> CJK * wphco}{}
+\DeclareFontShape{C63}{ph}{c}{n}{<-> CJKssub * ph/m/n}{}
+\DeclareFontShape{C63}{ph}{c}{sl}{<-> CJKssub * ph/b/n}{}
 \DeclareFontShape{C63}{ph}{c}{it}{<-> CJKssub * ph/c/sl}{}
 
 \DeclareFontShape{C63}{ph}{m}{n}{<-> CJK * wph}{}
-\DeclareFontShape{C63}{ph}{m}{sl}{<-> CJK * wpho}{}
+\DeclareFontShape{C63}{ph}{m}{sl}{<-> CJKssub * ph/b/n}{}
 \DeclareFontShape{C63}{ph}{m}{it}{<-> CJKssub * ph/m/sl}{}
 
-\DeclareFontShape{C63}{ph}{x}{n}{<-> CJK * wphx}{}
-\DeclareFontShape{C63}{ph}{x}{sl}{<-> CJK * wphxo}{}
+\DeclareFontShape{C63}{ph}{x}{n}{<-> CJKssub * ph/m/n}{}
+\DeclareFontShape{C63}{ph}{x}{sl}{<-> CJKssub * ph/b/n}{}
 \DeclareFontShape{C63}{ph}{x}{it}{<-> CJKssub * ph/x/sl}{}
 
-\DeclareFontShape{C63}{ph}{ux}{n}{<-> CJK * wphu}{}
-\DeclareFontShape{C63}{ph}{ux}{sl}{<-> CJK * wphuo}{}
+\DeclareFontShape{C63}{ph}{ux}{n}{<-> CJKssub * ph/m/n}{}
+\DeclareFontShape{C63}{ph}{ux}{sl}{<-> CJKssub * ph/b/n}{}
 \DeclareFontShape{C63}{ph}{ux}{it}{<-> CJKssub * ph/ux/sl}{}
 
-\DeclareFontShape{C63}{ph}{bc}{n}{<-> CJKb * wphc}{\CJKbold}
-\DeclareFontShape{C63}{ph}{bc}{sl}{<-> CJKb * wphco}{\CJKbold}
+\DeclareFontShape{C63}{ph}{bc}{n}{<-> CJKssub * ph/b/n}{}
+\DeclareFontShape{C63}{ph}{bc}{sl}{<-> CJKssub * ph/b/n}{}
 \DeclareFontShape{C63}{ph}{bc}{it}{<-> CJKssub * ph/bc/sl}{}
 
 \DeclareFontShape{C63}{ph}{b}{n}{<-> CJKb * wph}{\CJKbold}
-\DeclareFontShape{C63}{ph}{b}{sl}{<-> CJKb * wpho}{\CJKbold}
+\DeclareFontShape{C63}{ph}{b}{sl}{<-> CJKssub * ph/b/n}{}
 \DeclareFontShape{C63}{ph}{b}{it}{<-> CJKssub * ph/b/sl}{}
 
-\DeclareFontShape{C63}{ph}{bx}{n}{<-> CJKb * wphx}{\CJKbold}
-\DeclareFontShape{C63}{ph}{bx}{sl}{<-> CJKb * wphxo}{\CJKbold}
+\DeclareFontShape{C63}{ph}{bx}{n}{<-> CJKssub * ph/b/n}{}
+\DeclareFontShape{C63}{ph}{bx}{sl}{<-> CJKssub * ph/b/n}{}
 \DeclareFontShape{C63}{ph}{bx}{it}{<-> CJKssub * ph/bx/sl}{}
 
-\DeclareFontShape{C63}{ph}{bux}{n}{<-> CJKb * wphu}{\CJKbold}
-\DeclareFontShape{C63}{ph}{bux}{sl}{<-> CJKb * wphuo}{\CJKbold}
+\DeclareFontShape{C63}{ph}{bux}{n}{<-> CJKssub * ph/b/n}{}
+\DeclareFontShape{C63}{ph}{bux}{sl}{<-> CJKssub * ph/b/n}{}
 \DeclareFontShape{C63}{ph}{bux}{it}{<-> CJKssub * ph/bux/sl}{}
 
 \endinput
Index: cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63pn.fd
===================================================================
--- cjk-4.8.3+git20090105.orig/texinput/KS/HLaTeX/c63pn.fd	2010-05-17 01:03:13.699352847 +0200
+++ cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63pn.fd	2010-05-17 01:03:16.000000000 +0200
@@ -18,36 +18,36 @@
 
 \DeclareFontFamily{C63}{pn}{\hyphenchar \font\m@ne}
 
-\DeclareFontShape{C63}{pn}{c}{n}{<-> CJK * wpnc}{}
-\DeclareFontShape{C63}{pn}{c}{sl}{<-> CJK * wpnco}{}
+\DeclareFontShape{C63}{pn}{c}{n}{<-> CJKssub * pn/m/n}{}
+\DeclareFontShape{C63}{pn}{c}{sl}{<-> CJKssub * pn/b/n}{}
 \DeclareFontShape{C63}{pn}{c}{it}{<-> CJKssub * pn/c/sl}{}
 
 \DeclareFontShape{C63}{pn}{m}{n}{<-> CJK * wpn}{}
-\DeclareFontShape{C63}{pn}{m}{sl}{<-> CJK * wpno}{}
+\DeclareFontShape{C63}{pn}{m}{sl}{<-> CJKssub * pn/b/n}{}
 \DeclareFontShape{C63}{pn}{m}{it}{<-> CJKssub * pn/m/sl}{}
 
-\DeclareFontShape{C63}{pn}{x}{n}{<-> CJK * wpnx}{}
-\DeclareFontShape{C63}{pn}{x}{sl}{<-> CJK * wpnxo}{}
+\DeclareFontShape{C63}{pn}{x}{n}{<-> CJKssub * pn/m/n}{}
+\DeclareFontShape{C63}{pn}{x}{sl}{<-> CJKssub * pn/b/n}{}
 \DeclareFontShape{C63}{pn}{x}{it}{<-> CJKssub * pn/x/sl}{}
 
-\DeclareFontShape{C63}{pn}{ux}{n}{<-> CJK * wpnu}{}
-\DeclareFontShape{C63}{pn}{ux}{sl}{<-> CJK * wpnuo}{}
+\DeclareFontShape{C63}{pn}{ux}{n}{<-> CJKssub * pn/m/n}{}
+\DeclareFontShape{C63}{pn}{ux}{sl}{<-> CJKssub * pn/b/n}{}
 \DeclareFontShape{C63}{pn}{ux}{it}{<-> CJKssub * pn/ux/sl}{}
 
-\DeclareFontShape{C63}{pn}{bc}{n}{<-> CJKb * wpnc}{\CJKbold}
-\DeclareFontShape{C63}{pn}{bc}{sl}{<-> CJKb * wpnco}{\CJKbold}
+\DeclareFontShape{C63}{pn}{bc}{n}{<-> CJKssub * pn/b/n}{}
+\DeclareFontShape{C63}{pn}{bc}{sl}{<-> CJKssub * pn/b/n}{}
 \DeclareFontShape{C63}{pn}{bc}{it}{<-> CJKssub * pn/bc/sl}{}
 
 \DeclareFontShape{C63}{pn}{b}{n}{<-> CJKb * wpn}{\CJKbold}
-\DeclareFontShape{C63}{pn}{b}{sl}{<-> CJKb * wpno}{\CJKbold}
+\DeclareFontShape{C63}{pn}{b}{sl}{<-> CJKssub * pn/b/n}{}
 \DeclareFontShape{C63}{pn}{b}{it}{<-> CJKssub * pn/b/sl}{}
 
-\DeclareFontShape{C63}{pn}{bx}{n}{<-> CJKb * wpnx}{\CJKbold}
-\DeclareFontShape{C63}{pn}{bx}{sl}{<-> CJKb * wpnxo}{\CJKbold}
+\DeclareFontShape{C63}{pn}{bx}{n}{<-> CJKssub * pn/b/n}{}
+\DeclareFontShape{C63}{pn}{bx}{sl}{<-> CJKssub * pn/b/n}{}
 \DeclareFontShape{C63}{pn}{bx}{it}{<-> CJKssub * pn/bx/sl}{}
 
-\DeclareFontShape{C63}{pn}{bux}{n}{<-> CJKb * wpnu}{\CJKbold}
-\DeclareFontShape{C63}{pn}{bux}{sl}{<-> CJKb * wpnuo}{\CJKbold}
+\DeclareFontShape{C63}{pn}{bux}{n}{<-> CJKssub * pn/b/n}{}
+\DeclareFontShape{C63}{pn}{bux}{sl}{<-> CJKssub * pn/b/n}{}
 \DeclareFontShape{C63}{pn}{bux}{it}{<-> CJKssub * pn/bux/sl}{}
 
 \endinput
Index: cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63sh.fd
===================================================================
--- cjk-4.8.3+git20090105.orig/texinput/KS/HLaTeX/c63sh.fd	2010-05-17 01:03:13.707352786 +0200
+++ cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63sh.fd	2010-05-17 01:03:16.000000000 +0200
@@ -26,28 +26,28 @@
 \DeclareFontShape{C63}{sh}{m}{sl}{<-> CJK * wsho}{}
 \DeclareFontShape{C63}{sh}{m}{it}{<-> CJKssub * sh/m/sl}{}
 
-\DeclareFontShape{C63}{sh}{x}{n}{<-> CJK * wshx}{}
-\DeclareFontShape{C63}{sh}{x}{sl}{<-> CJK * wshxo}{}
+\DeclareFontShape{C63}{sh}{x}{n}{<-> CJKssub * sh/m/n}{}
+\DeclareFontShape{C63}{sh}{x}{sl}{<-> CJKssub * sh/m/sl}{}
 \DeclareFontShape{C63}{sh}{x}{it}{<-> CJKssub * sh/x/sl}{}
 
-\DeclareFontShape{C63}{sh}{ux}{n}{<-> CJK * wshu}{}
-\DeclareFontShape{C63}{sh}{ux}{sl}{<-> CJK * wshuo}{}
+\DeclareFontShape{C63}{sh}{ux}{n}{<-> CJKssub * sh/m/n}{}
+\DeclareFontShape{C63}{sh}{ux}{sl}{<-> CJKssub * sh/m/sl}{}
 \DeclareFontShape{C63}{sh}{ux}{it}{<-> CJKssub * sh/ux/sl}{}
 
-\DeclareFontShape{C63}{sh}{bc}{n}{<-> CJKssub * sh/c/n}{}
-\DeclareFontShape{C63}{sh}{bc}{sl}{<-> CJKssub * sh/c/sl}{}
+\DeclareFontShape{C63}{sh}{bc}{n}{<-> CJKssub * sh/b/n}{}
+\DeclareFontShape{C63}{sh}{bc}{sl}{<-> CJKssub * sh/b/sl}{}
 \DeclareFontShape{C63}{sh}{bc}{it}{<-> CJKssub * sh/bc/sl}{}
 
-\DeclareFontShape{C63}{sh}{b}{n}{<-> CJKssub * sh/m/n}{}
-\DeclareFontShape{C63}{sh}{b}{sl}{<-> CJKssub * sh/m/sl}{}
+\DeclareFontShape{C63}{sh}{b}{n}{<-> CJKb * wsh}{\CJKbold}
+\DeclareFontShape{C63}{sh}{b}{sl}{<-> CJKb * wsho}{\CJKbold}
 \DeclareFontShape{C63}{sh}{b}{it}{<-> CJKssub * sh/b/sl}{}
 
-\DeclareFontShape{C63}{sh}{bx}{n}{<-> CJKssub * sh/x/n}{}
-\DeclareFontShape{C63}{sh}{bx}{sl}{<-> CJKssub * sh/x/sl}{}
+\DeclareFontShape{C63}{sh}{bx}{n}{<-> CJKssub * sh/b/n}{}
+\DeclareFontShape{C63}{sh}{bx}{sl}{<-> CJKssub * sh/b/sl}{}
 \DeclareFontShape{C63}{sh}{bx}{it}{<-> CJKssub * sh/bx/sl}{}
 
-\DeclareFontShape{C63}{sh}{bux}{n}{<-> CJKssub * sh/ux/n}{}
-\DeclareFontShape{C63}{sh}{bux}{sl}{<-> CJKssub * sh/ux/sl}{}
+\DeclareFontShape{C63}{sh}{bux}{n}{<-> CJKssub * sh/b/n}{}
+\DeclareFontShape{C63}{sh}{bux}{sl}{<-> CJKssub * sh/b/sl}{}
 \DeclareFontShape{C63}{sh}{bux}{it}{<-> CJKssub * sh/bux/sl}{}
 
 \endinput
Index: cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63tt.fd
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63tt.fd	2010-05-17 01:03:16.000000000 +0200
@@ -0,0 +1,53 @@
+% This is file c63tt.fd of the CJK package
+%   for using Asian logographs (Chinese/Japanese/Korean) with LaTeX2e
+%
+% created by Werner Lemberg <wl@gnu.org>
+%
+% Version 4.8.3 (07-May-2012)
+
+\def\fileversion{4.8.3}
+\def\filedate{2008/12/29}
+\ProvidesFile{c63tt.fd}[\filedate\space\fileversion]
+
+% Korean characters (Hangul)
+%
+% character set: KS X 1001:1992 (=KS C 5601-1992)
+% font encoding: HL
+
+% UhcTypewriter
+
+\DeclareFontFamily{C63}{tt}{\hyphenchar \font\m@ne}
+
+\DeclareFontShape{C63}{tt}{c}{n}{<-> CJKssub * tt/m/n}{}
+\DeclareFontShape{C63}{tt}{c}{sl}{<-> CJKssub * tt/m/sl}{}
+\DeclareFontShape{C63}{tt}{c}{it}{<-> CJKssub * tt/c/sl}{}
+
+\DeclareFontShape{C63}{tt}{m}{n}{<-> CJK * wtt}{}
+\DeclareFontShape{C63}{tt}{m}{sl}{<-> CJK * wtto}{}
+\DeclareFontShape{C63}{tt}{m}{it}{<-> CJKssub * tt/m/sl}{}
+
+\DeclareFontShape{C63}{tt}{x}{n}{<-> CJKssub * tt/m/n}{}
+\DeclareFontShape{C63}{tt}{x}{sl}{<-> CJKssub * tt/m/sl}{}
+\DeclareFontShape{C63}{tt}{x}{it}{<-> CJKssub * tt/x/sl}{}
+
+\DeclareFontShape{C63}{tt}{ux}{n}{<-> CJKssub * tt/m/n}{}
+\DeclareFontShape{C63}{tt}{ux}{sl}{<-> CJKssub * tt/m/sl}{}
+\DeclareFontShape{C63}{tt}{ux}{it}{<-> CJKssub * tt/ux/sl}{}
+
+\DeclareFontShape{C63}{tt}{bc}{n}{<-> CJKssub * tt/b/n}{}
+\DeclareFontShape{C63}{tt}{bc}{sl}{<-> CJKssub * tt/b/sl}{}
+\DeclareFontShape{C63}{tt}{bc}{it}{<-> CJKssub * tt/bc/sl}{}
+
+\DeclareFontShape{C63}{tt}{b}{n}{<-> CJKb * wtt}{\CJKbold}
+\DeclareFontShape{C63}{tt}{b}{sl}{<-> CJKb * wtto}{\CJKbold}
+\DeclareFontShape{C63}{tt}{b}{it}{<-> CJKssub * tt/b/sl}{}
+
+\DeclareFontShape{C63}{tt}{bx}{n}{<-> CJKssub * tt/b/n}{}
+\DeclareFontShape{C63}{tt}{bx}{sl}{<-> CJKssub * tt/b/sl}{}
+\DeclareFontShape{C63}{tt}{bx}{it}{<-> CJKssub * tt/bx/sl}{}
+
+\DeclareFontShape{C63}{tt}{bux}{n}{<-> CJKssub * tt/b/n}{}
+\DeclareFontShape{C63}{tt}{bux}{sl}{<-> CJKssub * tt/b/sl}{}
+\DeclareFontShape{C63}{tt}{bux}{it}{<-> CJKssub * tt/bux/sl}{}
+
+\endinput
Index: cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63tz.fd
===================================================================
--- cjk-4.8.3+git20090105.orig/texinput/KS/HLaTeX/c63tz.fd	2010-05-17 01:03:13.723368309 +0200
+++ /dev/null	1970-01-01 00:00:00.000000000 +0000
@@ -1,53 +0,0 @@
-% This is file c63tz.fd of the CJK package
-%   for using Asian logographs (Chinese/Japanese/Korean) with LaTeX2e
-%
-% created by Werner Lemberg <wl@gnu.org>
-%
-% Version 4.8.3 (07-May-2012)
-
-\def\fileversion{4.8.3}
-\def\filedate{2012/05/07}
-\ProvidesFile{c63tz.fd}[\filedate\space\fileversion]
-
-% Korean characters (Hangul)
-%
-% character set: KS X 1001:1992 (=KS C 5601-1992)
-% font encoding: HL
-
-% UhcTypewriter
-
-\DeclareFontFamily{C63}{tz}{\hyphenchar \font\m@ne}
-
-\DeclareFontShape{C63}{tz}{c}{n}{<-> CJK * wttc}{}
-\DeclareFontShape{C63}{tz}{c}{sl}{<-> CJK * wttco}{}
-\DeclareFontShape{C63}{tz}{c}{it}{<-> CJKssub * tz/c/sl}{}
-
-\DeclareFontShape{C63}{tz}{m}{n}{<-> CJK * wtt}{}
-\DeclareFontShape{C63}{tz}{m}{sl}{<-> CJK * wtto}{}
-\DeclareFontShape{C63}{tz}{m}{it}{<-> CJKssub * tz/m/sl}{}
-
-\DeclareFontShape{C63}{tz}{x}{n}{<-> CJK * wttx}{}
-\DeclareFontShape{C63}{tz}{x}{sl}{<-> CJK * wttxo}{}
-\DeclareFontShape{C63}{tz}{x}{it}{<-> CJKssub * tz/x/sl}{}
-
-\DeclareFontShape{C63}{tz}{ux}{n}{<-> CJK * wttu}{}
-\DeclareFontShape{C63}{tz}{ux}{sl}{<-> CJK * wttuo}{}
-\DeclareFontShape{C63}{tz}{ux}{it}{<-> CJKssub * tz/ux/sl}{}
-
-\DeclareFontShape{C63}{tz}{bc}{n}{<-> CJKb * wttc}{\CJKbold}
-\DeclareFontShape{C63}{tz}{bc}{sl}{<-> CJKb * wttco}{\CJKbold}
-\DeclareFontShape{C63}{tz}{bc}{it}{<-> CJKssub * tz/bc/sl}{}
-
-\DeclareFontShape{C63}{tz}{b}{n}{<-> CJKb * wtt}{\CJKbold}
-\DeclareFontShape{C63}{tz}{b}{sl}{<-> CJKb * wtto}{\CJKbold}
-\DeclareFontShape{C63}{tz}{b}{it}{<-> CJKssub * tz/b/sl}{}
-
-\DeclareFontShape{C63}{tz}{bx}{n}{<-> CJKb * wttx}{\CJKbold}
-\DeclareFontShape{C63}{tz}{bx}{sl}{<-> CJKb * wttxo}{\CJKbold}
-\DeclareFontShape{C63}{tz}{bx}{it}{<-> CJKssub * tz/bx/sl}{}
-
-\DeclareFontShape{C63}{tz}{bux}{n}{<-> CJKb * wttu}{\CJKbold}
-\DeclareFontShape{C63}{tz}{bux}{sl}{<-> CJKb * wttuo}{\CJKbold}
-\DeclareFontShape{C63}{tz}{bux}{it}{<-> CJKssub * tz/bux/sl}{}
-
-\endinput
Index: cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63vd.fd
===================================================================
--- cjk-4.8.3+git20090105.orig/texinput/KS/HLaTeX/c63vd.fd	2010-05-17 01:03:13.731352813 +0200
+++ /dev/null	1970-01-01 00:00:00.000000000 +0000
@@ -1,53 +0,0 @@
-% This is file c63vd.fd of the CJK package
-%   for using Asian logographs (Chinese/Japanese/Korean) with LaTeX2e
-%
-% created by Werner Lemberg <wl@gnu.org>
-%
-% Version 4.8.3 (07-May-2012)
-
-\def\fileversion{4.8.3}
-\def\filedate{2012/05/07}
-\ProvidesFile{c63vd.fd}[\filedate\space\fileversion]
-
-% Korean characters (Hangul)
-%
-% character set: KS X 1001:1992 (=KS C 5601-1992)
-% font encoding: HL
-
-% UhcVada
-
-\DeclareFontFamily{C63}{vd}{\hyphenchar \font\m@ne}
-
-\DeclareFontShape{C63}{vd}{c}{n}{<-> CJK * wvdc}{}
-\DeclareFontShape{C63}{vd}{c}{sl}{<-> CJK * wvdco}{}
-\DeclareFontShape{C63}{vd}{c}{it}{<-> CJKssub * vd/c/sl}{}
-
-\DeclareFontShape{C63}{vd}{m}{n}{<-> CJK * wvd}{}
-\DeclareFontShape{C63}{vd}{m}{sl}{<-> CJK * wvdo}{}
-\DeclareFontShape{C63}{vd}{m}{it}{<-> CJKssub * vd/m/sl}{}
-
-\DeclareFontShape{C63}{vd}{x}{n}{<-> CJK * wvdx}{}
-\DeclareFontShape{C63}{vd}{x}{sl}{<-> CJK * wvdxo}{}
-\DeclareFontShape{C63}{vd}{x}{it}{<-> CJKssub * vd/x/sl}{}
-
-\DeclareFontShape{C63}{vd}{ux}{n}{<-> CJK * wvdu}{}
-\DeclareFontShape{C63}{vd}{ux}{sl}{<-> CJK * wvduo}{}
-\DeclareFontShape{C63}{vd}{ux}{it}{<-> CJKssub * vd/ux/sl}{}
-
-\DeclareFontShape{C63}{vd}{bc}{n}{<-> CJKb * wvdc}{\CJKbold}
-\DeclareFontShape{C63}{vd}{bc}{sl}{<-> CJKb * wvdco}{\CJKbold}
-\DeclareFontShape{C63}{vd}{bc}{it}{<-> CJKssub * vd/bc/sl}{}
-
-\DeclareFontShape{C63}{vd}{b}{n}{<-> CJKb * wvd}{\CJKbold}
-\DeclareFontShape{C63}{vd}{b}{sl}{<-> CJKb * wvdo}{\CJKbold}
-\DeclareFontShape{C63}{vd}{b}{it}{<-> CJKssub * vd/b/sl}{}
-
-\DeclareFontShape{C63}{vd}{bx}{n}{<-> CJKb * wvdx}{\CJKbold}
-\DeclareFontShape{C63}{vd}{bx}{sl}{<-> CJKb * wvdxo}{\CJKbold}
-\DeclareFontShape{C63}{vd}{bx}{it}{<-> CJKssub * vd/bx/sl}{}
-
-\DeclareFontShape{C63}{vd}{bux}{n}{<-> CJKb * wvdu}{\CJKbold}
-\DeclareFontShape{C63}{vd}{bux}{sl}{<-> CJKb * wvduo}{\CJKbold}
-\DeclareFontShape{C63}{vd}{bux}{it}{<-> CJKssub * vd/bux/sl}{}
-
-\endinput
Index: cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63yt.fd
===================================================================
--- cjk-4.8.3+git20090105.orig/texinput/KS/HLaTeX/c63yt.fd	2010-05-17 01:03:13.743352896 +0200
+++ cjk-4.8.3+git20090105/texinput/KS/HLaTeX/c63yt.fd	2010-05-17 01:03:16.000000000 +0200
@@ -26,12 +26,12 @@
 \DeclareFontShape{C63}{yt}{m}{sl}{<-> CJK * wyto}{}
 \DeclareFontShape{C63}{yt}{m}{it}{<-> CJKssub * yt/m/sl}{}
 
-\DeclareFontShape{C63}{yt}{x}{n}{<-> CJK * wytx}{}
-\DeclareFontShape{C63}{yt}{x}{sl}{<-> CJK * wytxo}{}
+\DeclareFontShape{C63}{yt}{x}{n}{<-> CJKssub * yt/m/n}{}
+\DeclareFontShape{C63}{yt}{x}{sl}{<-> CJKssub * yt/m/sl}{}
 \DeclareFontShape{C63}{yt}{x}{it}{<-> CJKssub * yt/x/sl}{}
 
-\DeclareFontShape{C63}{yt}{ux}{n}{<-> CJK * wytu}{}
-\DeclareFontShape{C63}{yt}{ux}{sl}{<-> CJK * wytuo}{}
+\DeclareFontShape{C63}{yt}{ux}{n}{<-> CJKssub * yt/m/n}{}
+\DeclareFontShape{C63}{yt}{ux}{sl}{<-> CJKssub * yt/m/sl}{}
 \DeclareFontShape{C63}{yt}{ux}{it}{<-> CJKssub * yt/ux/sl}{}
 
 \DeclareFontShape{C63}{yt}{bc}{n}{<-> CJKb * wytc}{\CJKbold}
@@ -42,12 +42,12 @@
 \DeclareFontShape{C63}{yt}{b}{sl}{<-> CJKb * wyto}{\CJKbold}
 \DeclareFontShape{C63}{yt}{b}{it}{<-> CJKssub * yt/b/sl}{}
 
-\DeclareFontShape{C63}{yt}{bx}{n}{<-> CJKb * wytx}{\CJKbold}
-\DeclareFontShape{C63}{yt}{bx}{sl}{<-> CJKb * wytxo}{\CJKbold}
+\DeclareFontShape{C63}{yt}{bx}{n}{<-> CJKssub * yt/b/n}{}
+\DeclareFontShape{C63}{yt}{bx}{sl}{<-> CJKssub * yt/b/sl}{}
 \DeclareFontShape{C63}{yt}{bx}{it}{<-> CJKssub * yt/bx/sl}{}
 
-\DeclareFontShape{C63}{yt}{bux}{n}{<-> CJKb * wytu}{\CJKbold}
-\DeclareFontShape{C63}{yt}{bux}{sl}{<-> CJKb * wytuo}{\CJKbold}
+\DeclareFontShape{C63}{yt}{bux}{n}{<-> CJKssub * yt/b/n}{}
+\DeclareFontShape{C63}{yt}{bux}{sl}{<-> CJKssub * yt/b/sl}{}
 \DeclareFontShape{C63}{yt}{bux}{it}{<-> CJKssub * yt/bux/sl}{}
 
 \endinput
